tests/test_field_handler.py

```diff
@@ -6,7 +6,7 @@
                              HasManyDescriptor, HasAndBelongsToManyDescriptor)
 from remodel.utils import create_tables, create_indexes
 
-from . import BaseTestCase, DbBaseTestCase
```

Comment:
This seems to be working on Python 3, why have you changed the relative import? (same note for similar cases below)